src/app/core/services/scheduling.service.ts

```diff
@@ -370,8 +324,13 @@ export class SchedulingService {
     return amount * TIME_UNIT_MILLIS[unit]
```

Comment:
In this function, can we make the default unit and the default amount as a default config? Or at least make it a constant at the top of the service?